src/hearings/containers/request-hearing/hearing-view-edit-summary/hearing-view-edit-summary.component.ts

```diff
@@ -12,6 +12,7 @@ import { HEARING_VIEW_EDIT_SUMMARY_TEMPLATE } from '../../../templates/hearing-v
 import { RequestHearingPageFlow } from '../request-hearing.page.flow';
 import { Section } from '../../../../hearings/models/section';
 import { ScreenNavigationModel } from '../../../../hearings/models/screenNavigation.model';
+import { HearingResponseError } from 'src/hearings/models/hearingResponseError.model';
```

Comment:
need to remove reference to src, should be ../../../../

Reply:
Removed ref to src